redbot/__main__.py

```diff
@@ -401,6 +401,24 @@ async def run_bot(red: Red, cli_flags: Namespace) -> None:
             "https://docs.discord.red/en/stable/bot_application_guide.html#enabling-privileged-intents"
         )
         sys.exit(1)
+    except _NoOwnerSet:
+        print(
+            "Bot doesn't have any owner set!\n"
+            "This can happen when your bot's application is owned by team"
+            " as team members are NOT owners by default.\n\n"
+            "Remember:\n"
+            "ONLY the person who is hosting Red should be owner."
+            " This has SERIOUS security implications."
+            " The owner can access any data that is present on the host system.\n"
+            "With that out of the way, depending on who you want to be considered as owner,"
+            " you can:\n"
+            "a) pass --team-members-are-owners when launching Red"
+            " - in this case Red will treat all members of the bot application's team as owners\n"
+            "b) set owner manually with redbot --edit <instance_name>\n"
```

Comment:
Probably better would be enclose `redbot --edit <instance_name>` in quotes

Reply:
Done.